client/i18n/locales/english/intro.json

```diff
@@ -1822,7 +1822,12 @@
       "uqiy": { "title": "67", "intro": [] },
       "jnqt": { "title": "68", "intro": [] },
       "vkei": { "title": "69", "intro": [] },
-      "jows": { "title": "70", "intro": [] },
+      "lab-contact-form": {
+        "title": "Build a Contact Form",
```

Comment:
maybe `Design a Contact Form` will be better here.

if we decide on the name change, then it should be updated in all places